VERSION

```diff
@@ -1 +1 @@
-0.4.0
+0.4.1
```

Comment:
can you the trailing newline that's currently in this file?

Reply:
yep, done